samples/storage_csm_powerflex_v2101.yaml

```diff
@@ -18,6 +18,9 @@ spec:
       #   false: disable storage capacity tracking
       storageCapacity: true
     configVersion: v2.10.1
+    # Controller count
+    # Allowed values: n, where n > 0
+    # Default value: None
```

Comment:
Default value cannot be none here, it has to be the value that will be created if you don't provide one